Code/Framework/AzFramework/AzFramework/Physics/CharacterBus.h

```diff
@@ -103,4 +104,75 @@ namespace Physics
     };
 
     using CharacterRequestBus = AZ::EBus<CharacterRequests>;
-} // namespace Physics
+
+    /// Messages sent by character controllers.
```

Comment:
uber nit: I believe `//!` style is generally preferred now if possible

Reply:
This entire file is using /// so I'd rather keep it consistent and not do unnecessary changes in other places now.